kobo/apps/subsequences/models.py

```diff
@@ -34,124 +28,21 @@ class Meta:
         unique_together = (('asset', 'submission_uuid'),)
 
     def save(self, *args, **kwargs):
-        features = self.asset.advanced_features
-        if 'transcript' in features:
-            for qpath, vals in self.content.items():
-                try:
-                    autoparams = vals[GOOGLETS]
-                    status = autoparams['status']
-                    if status == 'requested':
-                        username = self.asset.owner.username
-                        engine = GoogleTranscribeEngine()
-                        service = TranscriptionService.objects.get(code='goog')
-                        language_code = autoparams.get('languageCode')
-                        region_code = autoparams.get('regionCode')
-                        vals[GOOGLETS] = {
-                            'status': 'in_progress',
-                            'languageCode': language_code,
-                            'regionCode': region_code,
-                        }
-                        for row in self.asset.content['survey']:
-                            if '$qpath' in row and '$xpath' in row:
-                                if row['$qpath'] == qpath:
-                                    xpath = row['$xpath']
-                                    break
-                        region_or_language_code = region_code or language_code
-                        try:
-                            results = engine.transcribe_file(
-                                asset=self.asset,
-                                xpath=xpath,
-                                source=region_or_language_code,
-                                submission_id=self.submission_uuid,
-                                user=self.asset.owner,
-                            )
-                        except SubsequenceTimeoutError:
-                            continue
-                        except TranscriptionResultsNotFound:
-                            logging.error(f'No transcriptions found for {xpath}')
-                            result_string = ''
-                            results = []
-                        else:
-                            result_string = ' '.join(
-                                [r['transcript'] for r in results]
-                            )
-
-                        vals[GOOGLETS] = {
-                            'status': 'complete',
-                            'value': result_string,
-                            'fullResponse': results,
-                            'languageCode': language_code,
-                            'regionCode': region_code,
-                        }
-                    else:
-                        continue
-                except (KeyError, TypeError) as err:
-                    continue
+        from .integrations.google.google_transcribe import GoogleTranscriptionService
+        from .integrations.google.google_translate import GoogleTranslationService
 
```

Comment:
Why here? Because of circular imports? This could not be avoided? 
Please add a comment if not.

Reply:
ok